src/services/babel-browser.gen.js

```diff
@@ -0,0 +1,77 @@
+const babel = require('@babel/core');
+const presetEnv = require('@babel/preset-env');
+
+const browserList = require('../../config/browsers.json');
+
+const EXCLUDED_MODULES = ['transform-regenerator'];
+
+function getPluginNamesFromPresetEnv() {
+  const browserListForBabel = {};
+
+  for (const browser in browserList) {
+    if (browser !== 'chromium') {
+      browserListForBabel[browser] = browserList[browser];
+    }
+  }
+
+  const result = babel.transformSync('', {
+    sourceType: 'script',
+    presets: [
+      [
+        presetEnv,
+        {
+          targets: browserListForBabel,
+          exclude: EXCLUDED_MODULES,
+        },
+      ],
+    ],
+  });
+
+  return result.options.plugins
+    .map(plugin => plugin.key)
+    .filter(key => key.indexOf('/') !== 0);
+}
+
+module.exports = () => {
+  const keys = getPluginNamesFromPresetEnv();
+
+  const pluginImports = keys
+    .map((key, index) => `import plugin${index} from\n'@babel/plugin-${key}';`)
+    .join('\n');
+
+  const pluginNames = keys.map((key, index) => `plugin${index}`).join(',\n');
+
+  return {
+    code: `
```

Comment:
It seems mildly preferable to split this into two files—one of which, the `.gen.js`, just exports the list of plugin names; and the other which imports it and contains everything below. That way we avoid having substantial code in a string literal.